org.okstar.ok-msg.yml

```diff
@@ -0,0 +1,108 @@
+id: org.okstar.ok-msg
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: ok-msg-desktop
+rename-icon: ok-msg
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=all
+  - --talk-name=org.gnome.Mutter.IdleMonitor
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.indicator.application
+  - --talk-name=org.ayatana.indicator.application
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=xdg-documents
+  - --unset-env=QT_PLUGIN_PATH
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: '23.08'
+    autodownload: true
+    autodelete: false
+
+build-options:
+  cflags: -DNDEBUG
+  cxxflags: -DNDEBUG
+  build-args:
+    - --share=ipc
+    - --share=network
```

Comment:
```suggestion
```

Reply:
Because  at building need to access network to download dependencies , otherwise cannot  pass the build.

Reply:
Then it won't build, Flathub builds without network access.

Reply:
At building cmake need to access network to download dependencies , otherwise cannot  pass the build.

Reply:
Again, this is not possible and it will fail. You need to download the dependencies ahead of time by having them listed in the manifest.